products/datacatalog/api.yaml

```diff
@@ -487,3 +490,122 @@ objects:
 
           For attaching a tag to a nested column, use `.` to separate the column names. Example:
           `outer_column.inner_column`
+  - !ruby/object:Api::Resource
+    name: Taxonomy
+    base_url: projects/{{project}}/locations/{{region}}/taxonomies
+    self_link: "{{name}}"
+    min_version: beta
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      A collection of policy tags that classify data along a common axis.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': https://cloud.google.com/data-catalog/docs
+      api: https://cloud.google.com/data-catalog/docs/reference/rest/v1beta1/projects.locations.taxonomies
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      method_name_separator: ':'
+      fetch_iam_policy_verb: :POST
+      parent_resource_attribute: 'taxonomy'
+      import_format: ["projects/{{project}}/locations/{{region}}/taxonomies/{{taxonomy}}", "{{taxonomy}}"]
+      base_url: projects/{{project}}/locations/{{region}}/taxonomies/{{taxonomy}}
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: region
+        url_param_only: true
+        input: true
+        description: |
+          Taxonomy location region. Regions currently supported: eu, us, asia-northeast3
+
+          Regions supported after GA launch: eu, us, asia_east1, asia_east2, asia_south1, asia_southeast1, asia_southeast2, asia_northeast1, asia_northeast2, asia_northeast3, autralia_southeast1, europe_north1, europe_west1, europe_west2, europe_west3, europe_west4, europe_west6, northamerica_northeast1, southamerica_east1, us_east1, us_east4, us_west1, us_west2, us_west3, us_west4
```

Comment:
We don't usually include these details in the region field - and I think actually we might want to rename this field location, since these are not *quite* regions.  "eu" and "us", at least.

Reply:
Keeping as is to align with other resources from this product group